src/main/java/com/eternalcode/combat/fight/FightManager.java

```diff
@@ -23,10 +33,12 @@ public boolean isInCombat(UUID player) {
     }
 
     public void untag(UUID player) {
+        this.eventCaller.callEvent(new FightUntagEvent(Bukkit.getPlayer(player)));
         this.fights.remove(player);
     }
 
     public void tag(UUID target, Duration delay) {
+        this.eventCaller.callEvent(new FightTagEvent(Bukkit.getPlayer(target)));
```

Comment:
^
```suggestion
        this.eventCaller.callEvent(new FightTagEvent(Bukkit.getPlayer(target)));
        
```

Reply:
I have changed to UUID arguments inside Tag / UnTag events.